CHANGELOG.md

```diff
@@ -13,6 +13,8 @@
   - snowflake.snowpark.functions
     - udf
 - Added the function `DataFrame.write.csv` to unload data from a ``DataFrame`` into one or more CSV files in a stage.
+- Added the function `DataFrame.write.json` to unload data from a ``DataFrame`` into one or more JSON files in a stage.
+- Added the function `DataFrame.write.parquet` to unload data from a ``DataFrame`` into one or more PARQUET files in a stage.
```

Comment:
this can be combined into one
```suggestion
- Added the functions below to unload data from a `DataFrame` into one or more files in a stage:
  - `DataFrame.write.json`
  - `DataFrame.write.csv`
  - `DataFrame.write.parquet`
```